src/Symfony/Bundle/SecurityBundle/Security/FirewallConfig.php

```diff
@@ -29,21 +29,7 @@
     private $listeners;
     private $switchUser;
 
-    /**
-     * @param string      $name
-     * @param string      $userChecker
-     * @param string|null $requestMatcher
-     * @param bool        $securityEnabled
-     * @param bool        $stateless
-     * @param string|null $provider
-     * @param string|null $context
-     * @param string|null $entryPoint
-     * @param string|null $accessDeniedHandler
-     * @param string|null $accessDeniedUrl
-     * @param string[]    $listeners
-     * @param array|null  $switchUser
-     */
-    public function __construct($name, $userChecker, $requestMatcher = null, $securityEnabled = true, $stateless = false, $provider = null, $context = null, $entryPoint = null, $accessDeniedHandler = null, $accessDeniedUrl = null, $listeners = array(), $switchUser = null)
+    public function __construct(string $name, string $userChecker, string $requestMatcher = null, bool $securityEnabled = true, bool $stateless = false, string $provider = null, string $context = null, string $entryPoint = null, string $accessDeniedHandler = null, string $accessDeniedUrl = null, array $listeners = array(), $switchUser = null)
```

Comment:
Just noticed https://github.com/symfony/symfony/issues/12456 so think the consensus is to keep old style syntax, unless opinions have changed?